<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="vlMain">
   <item>
    <widget class="QWidget" name="shadowWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#shadowWidget
{
background-color: rgb(255, 255, 255);
border-radius:5px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑 Light</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Frameless</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mixBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border:none;
}

QPushButton:hover
{
	
	background-color: rgb(146, 181, 255);
}
QPushButton:pressed{
	
	background-color: rgb(93, 116, 162);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="maxBtn">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border:none;
}

QPushButton:hover
{
	
	background-color: rgb(146, 181, 255);
}
QPushButton:pressed{
	
	background-color: rgb(93, 116, 162);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitBtn">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(255, 0, 0);
border-top-right-radius:5px;
}

QPushButton:hover
{
	
	background-color: rgb(255, 79, 67);
}
QPushButton:pressed{
	
	background-color: rgb(255, 30, 1);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="maximumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnGetFile">
             <property name="text">
              <string>getFile</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QListWidget" name="listWidgetMd5"/>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetStr"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
